js/classes/entities/midground/chad.js

```diff
@@ -64,6 +64,8 @@ class Chad {
         this.isJumping = false;
         /** The timer for the jump. Used to ensure the jump force is applied for a minimum amount of time. */
         this.firstJumpTimer = 0;
+        this.dashTimer = 0;
```

Comment:
Do you mind taking out the dash timer? I implemented a dash timer a while ago on my fork called `this.groundDashTimer`. It should already be on the merge branch unless something weird happened in GitHub.

Reply:
Interesting. It wasn't there when I was working on the merge code, perhaps it was outdated. Sure thing, I'll update it.